From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 19 Sep 2012 10:22:39 -0400
Subject: look for base config in /etc/epics/configure

When building external modules (missing $(TOP)/configure/CONFIG_BASE_VERSION))
leave EPICS_BASE=/usr/lib/epics, but take configuration from
/etc/epics/configure.  This gives sysadmins the option to
modify build config.
---
 configure/CONFIG | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

Index: epics-base/configure/CONFIG
===================================================================
--- epics-base.orig/configure/CONFIG
+++ epics-base/configure/CONFIG
@@ -16,11 +16,11 @@ ifneq ($(wildcard $(TOP)/configure/CONFI
   CONFIG = $(TOP)/configure
   BASE_TOP=YES
 else
-  ifneq ($(origin EPICS_BASE),file)
-    # Essential for the EPICS build system, see convertRelease.pl
-    $(error EPICS_BASE must be set in a configure/RELEASE file)
+  ifeq ($(CONFIG),/usr/lib/epics/configure)
+    CONFIG = /etc/epics/configure
+  else
+    CONFIG ?= /etc/epics/configure
   endif
-  CONFIG ?= $(EPICS_BASE)/configure
 endif
 
 # Provide a default if the user hasn't set EPICS_HOST_ARCH
